recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend

```diff
@@ -1,4 +1,4 @@
 RENESAS_BSP_URL = "git://github.com/xen-troops/linux.git"
 
-BRANCH = "master"
+BRANCH = "v4.14-rcar-3.7.0-xt0.1"
```

Comment:
Why branch is named 3.7.0?

Reply:
Yocto-3.9.0 = BSP 3.7.0
Yocto-3.7.0 = BSP 3.6.2

Reply:
so that is a temporary development branch name, which name is actually not important